Foundation/FileManager.swift

```diff
@@ -442,16 +442,34 @@ open class FileManager : NSObject {
             else {
                 return
         }
+
+        func copyNonDirectory(srcPath: String, dstPath: String, fileType: FileAttributeType) throws {
+            if fileType == .typeSymbolicLink {
+                let destination = try destinationOfSymbolicLink(atPath: srcPath)
+                try createSymbolicLink(atPath: dstPath, withDestinationPath: destination)
+            } else if fileType == .typeRegular {
+                if createFile(atPath: dstPath, contents: contents(atPath: srcPath), attributes: nil) == false {
+                    throw NSError(domain: NSCocoaErrorDomain, code: CocoaError.fileWriteUnknown.rawValue, userInfo: [NSFilePathErrorKey : NSString(string: dstPath)])
+                }
+            }
+        }
+
         if fileType == .typeDirectory {
             try createDirectory(atPath: dstPath, withIntermediateDirectories: false, attributes: nil)
             let subpaths = try subpathsOfDirectory(atPath: srcPath)
             for subpath in subpaths {
-                try copyItem(atPath: srcPath + "/" + subpath, toPath: dstPath + "/" + subpath)
+                let src = srcPath + "/" + subpath
+                let dst = dstPath + "/" + subpath
```

Comment:
I'm sorry, I don't recall—Do we not have an equivalent of `-stringByAppendingPathComponent:` in SCL?

Reply:
Only on `NSString` it appears, maybe a helper on `String` could be added in `NSPathUtilities`.